chrome/locale/en-US/zotero/preferences.ftl

```diff
@@ -36,3 +36,11 @@ preferences-color-scheme-dark =
 preferences-advanced-language-and-region-title = Language and Region
 preferences-advanced-enable-bidi-ui =
     .label = Enable bidirectional text editing utilities
+preferences-quickCopy-plus =
+    .aria-label = { general-add }
+    .label = { $label }
+preferences-quickCopy-minus =
+    .aria-label = { general-remove }
+    .label = { $label }
+preferences-general-locate-resolver = Resolver
+preferences-general-locate-base-url = Base URL
```

Comment:
Let's actually remove the `general-` here — I know we use `advanced-` in a couple above, but better to have only the section (`locate-`) so that the id doesn't get out of sync if we decide to move this to another tab.

Reply:
That makes sense - I renamed these